dev/tests/integration/testsuite/Magento/Sales/Model/Order/StatusTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Sales\Model\Order;
+
+/**
+ * Class ShipmentTest
+ * @package Magento\Sales\Model\Order
+ */
+class StatusTest extends \PHPUnit\Framework\TestCase
```

Comment:
you can have I test + dataPorviderer instead of 2 similar tests

Reply:
@okorshenko are you happy with this updated test?